src/types/coursDetails.type.ts

```diff
@@ -0,0 +1,9 @@
+export type coursDetails = {
+    id: number,
+    title: string,
+    level: string,
```

Comment:
I recommend using enum to avoid mistakes 
```suggestion
export enum CourseLevel {
  EASY = 'easy',
  MEDIUM = 'medium',
  HARD = 'hard',
}
...
    level: CourseLevel,
```

Reply:
ok i will work on that